File: src/main/java/com/github/dkurata38/open_web_library/web/config/WebConfig.java
package com.github.dkurata38.open_web_library.web.config;

import org.springframework.boot.web.client.RestTemplateBuilder;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.client.RestTemplate;

@Configuration
public class WebConfig {
	@Bean
	public RestTemplate restTemplate () {
		return new RestTemplateBuilder().build();
	}
}

File: web/src/main/kotlin/com/github/dkurata38/open_web_library/web/WebConfig.kt
package com.github.dkurata38.open_web_library.web

import com.github.dkurata38.open_web_library.web.converters.ISBNConverter
import org.springframework.context.annotation.Configuration
import org.springframework.format.FormatterRegistry
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer

@Configuration
class WebConfig : WebMvcConfigurer {
	override fun addFormatters(registry: FormatterRegistry) {
		registry.addConverter(ISBNConverter())
	}
}
